models/eprop_learning_signal_connection.h

```diff
@@ -0,0 +1,233 @@
+/*
+ *  eprop_learning_signal_connection.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+#ifndef EPROP_LEARNING_SIGNAL_CONNECTION_H
+#define EPROP_LEARNING_SIGNAL_CONNECTION_H
+
+// nestkernel
+#include "connection.h"
+
+namespace nest
+{
+
+/* BeginUserDocs: synapse, e-prop plasticity
+
+Short description
++++++++++++++++++
+
+Synapse model transmitting feedback learning signals for e-prop plasticity
+
+Description
++++++++++++
+
+``eprop_learning_signal_connection`` is an implementation of a feedback connector from
+``eprop_readout_bsshslm_2020`` readout neurons to ``eprop_iaf_bsshslm_2020`` or ``eprop_iaf_adapt_bsshslm_2020``
+recurrent neurons that transmits the learning signals :math:`L_j^t` for eligibility propagation (e-prop) plasticity and
+has a static weight :math:`B_{jk}`.
+
+E-prop plasticity was originally introduced and implemented in TensorFlow in [1]_.
+
+The suffix ``_bsshslm_2020`` follows the NEST convention to indicate in the
```

Comment:
suffix should be removed

Reply:
Fixed in https://github.com/nest/nest-simulator/pull/3207/commits/6eda9de0b31a9dd88816ada412d8d5649c145b9e